lib/managers/window_manager.cpp

```diff
@@ -0,0 +1,47 @@
+#include "window_manager.h"
+
+#include "lib/logger/logger.h"
+
+std::vector<GLFWwindow*> WindowManager::windows_ = {};
+WindowID WindowManager::active_window_id_ = 0;
+
+GLFWwindow* WindowManager::get_active_window() {
+    return windows_.at(active_window_id_);
```

Comment:
Aaand what if we will want to add the ability to close previously opened windows?
The indexation system will break.